src/vs/workbench/workbench.common.main.ts

```diff
@@ -249,6 +249,9 @@ import 'vs/workbench/contrib/tasks/browser/task.contribution';
 import 'vs/workbench/contrib/remote/common/remote.contribution';
 import 'vs/workbench/contrib/remote/browser/remote.contribution';
 
+// Dnd
+import 'vs/workbench/contrib/dropIntoEditor/browser/dnd.contribution';
```

Comment:
Maybe it should be `dropIntoEditor.contribution` to be in sync?

Also, I wonder if this should be core and not contributed. Seems like a core functionality to me in a way.